backend/db/migrate/20220518133729_add_impacts_to_projects.rb

```diff
@@ -0,0 +1,19 @@
+class AddImpactsToProjects < ActiveRecord::Migration[7.0]
+  def change
+    add_column :projects, :municipality_biodiversity_impact, :decimal, precision: 25, scale: 20
```

Comment:
I'm not sure what was the current plan, but at the beginning we designed impacts to be in a separate table (and I think that is on DB design diagram), as those changes over time. There was a plan that was removed from the scope of MVP to be able to draw impact history timeline, but as I remember, the project should keep old impacts if they wanted to add the timeline in the second phase. I might be mistaken. cc @agnessa

Reply:
I would personally keep it simple at this phase and moved it to separate table when we will know more at future -- otherwise it will add extra complexity to sorting and calculation service at this point, which is not necessary right now